.identity/.gitignore

```diff
@@ -0,0 +1,48 @@
+# Local .terraform directories
```

Comment:
Please, move the entire configuration under `infra/identity`, we are gradually removing the `.identity` folder at the root